cli/cmd/env.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/env"
+	"github.com/tarantool/tt/cli/modules"
+)
+
+// NewEnvCmd creates env command.
+func NewEnvCmd() *cobra.Command {
+	var envCmd = &cobra.Command{
+		Use:   "env",
+		Short: "Add 'tarantool' and 'tt' (installed using 'tt') to the current environment.",
```

Comment:
It think we just add `tt` binary dir to the PATH.

Reply:
Kind of, but we also set TARANTOOL_DIR variable

Reply:
ok, changed